phone-store/package.json

```diff
@@ -7,7 +7,9 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "pree2e": "node node_modules/protractor/bin/webdriver-manager -- update --gecko false",
+    "e2eci": "ng e2e --protractor-config=e2e/protractor-ci.conf.js"
```

Comment:
Is this just here to provide a way call `ng e2e` with the right config file?

Was there a reason to add this here instead of just adding the flag to the `e2e-tests` job up above? (I'm not sure I care either way – just curious.)

Reply:
Yes this could be just copied into the GH Actions workflow instead of having it here. That might be better since it would be consistent with the unit tests. This is just how it was in the iteration template.

Reply:
I'm not sure I particularly care, but I did want to make sure I understood.

It does seem semi-odd to introduce the new `e2eci` task when we could just use the additional flag, though. Is there any setting outside of GitHub Actions where one might want to use `ng e2eci`? If so, then that might justify leaving it in. One could run `ng e2eci` on a droplet, for example, to make sure all the pieces work there – would that be potentially useful?